dev/dependencyList

```diff
@@ -29,6 +29,8 @@ htrace-core4/4.1.0-incubating//htrace-core4-4.1.0-incubating.jar
 jackson-annotations/2.11.4//jackson-annotations-2.11.4.jar
 jackson-core/2.11.4//jackson-core-2.11.4.jar
 jackson-databind/2.11.4//jackson-databind-2.11.4.jar
+jackson-module-paranamer/2.11.4//jackson-module-paranamer-2.11.4.jar
```

Comment:
We need to update LICENSE-binary once `dependencyList` changed

Reply:
Thanks for reviewing, done.

Reply:
Please open the file `LICENSE-binary` and scroll down to the bottom, we also need to list the deps there.